h5pxblock/h5pxblock.py

```diff
@@ -445,7 +445,8 @@ def is_past_due(self):
         """
         if not self.due:
             return False
-        return datetime.now() > self.due
+        due_tz = self.due.tzinfo
+        return datetime.now(due_tz) > self.due
```

Comment:
I'm not sure why you need a single use variable `due_tz ` here. Can we do it like this instead?
```suggestion
        return datetime.now(self.due.tzinfo) > self.due
```